src/main/java/nextstep/courses/domain/Session.java

```diff
@@ -0,0 +1,70 @@
+package nextstep.courses.domain;
+
+import nextstep.payments.domain.Payment;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Session {
+    private Long sessionId;
+
+    private SessionPeriod sessionPeriod;
+
+    private SessionCoverImage sessionCoverImage;
+
+    private SessionFeeType sessionFeeType;
+
+    private Long amount;
+
+    private Integer maxPersonnel;
+
+    private SessionStatus sessionStatus;
```

Comment:
```suggestion
    private Long id;

    private SessionPeriod period;

    private SessionCoverImage coverImage;

    private SessionFeeType feeType;

    private Long amount;

    private Integer maxPersonnel;

    private SessionStatus status;
```
Session 내 필드이므로 session prefix는 제거해도 좋을 것 같습니다.

`session.sessionId` 같이 중복 의미가 발생할 수 있습니다.

Reply:
prefix 제거 했습니다!